scripts/20230826_unset-encryption-boundary-shift/unset-encryption-boundary-shift.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable */
```

Comment:
qn: do we still need this script?

Reply:
As discussed, yes, it's a clean up script for the form-level `encryptionBoundaryShift` flags, if any.